tests/integration/components/hearings-list-for-milestones-list-test.js

```diff
@@ -0,0 +1,22 @@
+import { module, skip } from 'qunit';
+import { setupRenderingTest } from 'ember-qunit';
+import { render } from '@ember/test-helpers';
+import hbs from 'htmlbars-inline-precompile';
+
+module('Integration | Component | hearings-list-for-milestones-list', function(hooks) {
+  setupRenderingTest(hooks);
+
+  skip('it renders', async function(assert) {
```

Comment:
🤔

Reply:
Did this for the sake of time considering the demo is tomorrow and it was already 7pm -- was going to fix in another refactor where I could focus on creating a more detailed integration test like some of the other components. Currently have an acceptance test that checks for this component's rendering in all three tabs